components/ai_chat/resources/page/components/tools_button_menu/index.tsx

```diff
@@ -30,7 +28,7 @@ export default function ToolsButtonMenu(props: Props) {
       isOpen={props.isToolsMenuOpen}
       onClose={() => props.setIsToolsMenuOpen(false)}
     >
-      <div slot='anchor-content'>{props.children}</div>
+      <div slot='anchor-content' />
```

Comment:
do we still need this @Douglashdaniel?

Reply:
removed that in 2a2ac9ac546ecc612c6726cd1b233d6a90e7c0bc